index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h3>Welcome</h3>
+      <img src="./images/Union.svg" alt="menu icon" />
+    </header>
+    <section class="headline">
+      <div class="whoiam">
+        <h2 class="name">Hey there. I'm Fatima Ezzahra</h2>
+        <h2 class="job">I'm a software developer</h2>
+      </div>
+      <div class="aboutme">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social_media_icons">
+        <a href="#"><img src="./images/twitter.svg" alt="twitter" /></a>
+        <a href="#"><img src="./images/Linkedin.svg" alt="linkedin" /></a>
+        <a href="#"><img src="./images/medium.svg" alt="medium" /></a>
+        <a href="#"><img src="./images/github.svg" alt="github" /></a>
+        <a href="#"><img src="./images/angellist.svg" alt="angellist" /></a>
+      </div>
```

Comment:
- [ ] Again with semantic HTML, kindly change this div to a `<ul></ul>` and wrap each one of the items in a `<li></li>` tag, one of the goals with the HTML and CSS module is that soon you'll be writing semantic HTML without even thinking about it 😃.